common/src/lib.rs

```diff
@@ -787,4 +803,343 @@ mod tests {
             assert_eq!(get_code(program.code_hash).unwrap(), code);
         });
     }
+
+    #[test]
+    fn pause_program_works() {
+        sp_io::TestExternalities::new_empty().execute_with(|| {
+            let code = b"pretended wasm code".to_vec();
+            let code_hash: H256 = sp_io::hashing::blake2_256(&code[..]).into();
+            set_code(code_hash, &code);
+
+            let static_pages = 16;
+            let memory_pages = {
+                let mut pages = BTreeMap::<u32, Vec<u8>>::new();
```

Comment:
Isn't better to not point generic types explicitly?
```suggestion
            let static_pages: u32 = 16;
            let memory_pages = {
                let mut pages = BTreeMap::new();
```